nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -0,0 +1,184 @@
+{ config, lib, pkgs, buildEnv, ... }:
+
+with lib;
+
+let
+  cfg = config.services.healthchecks;
+  pkg = pkgs.healthchecks;
+  boolToPython = b: if b then "True" else "False";
+  environment = {
+    PYTHONPATH = pkg.pythonPath;
+    STATIC_ROOT = cfg.dataDir + "/static";
+    SECRET_KEY_FILE = cfg.secretKeyFile;
+    DB_NAME = "${cfg.dataDir}/healthchecks.sqlite";
+    ALLOWED_HOSTS = lib.concatStringsSep "," cfg.allowedHosts;
+    DEBUG = boolToPython cfg.debug;
+    REGISTRATION_OPEN = boolToPython cfg.registrationOpen;
+  } // cfg.extraConfig;
+
+  environmentFile = pkgs.writeText "healthchecks-environment" (lib.generators.toKeyValue {} environment);
+
+  healthchecksManageScript = with pkgs; (writeShellScriptBin "healthchecks" ''
+    sudo -u healthchecks sh <<EOF
+      export $(cat ${environmentFile} | xargs);
+      ${pkg}/bin/healthchecks $@
```

Comment:
```suggestion
      ${pkg}/bin/healthchecks "$@"
```

Reply:
This wouldn't work as expected as multiple command line args to the script would be passed as a single string. e.g.

Current: 
```sh
$ healthchecks help check
usage: manage.py check [-h] [--tag TAGS] [--list-tags] [--deploy] [--fail-level {CRITICAL,ERROR,WARNING,INFO,DEBUG}] [--database DATABASES] [--version] [-v {0,1,2,3}] [--settings SETTINGS] [--pythonpath PYTHONPATH] [--traceback] [--no-color] [--force-color] [app_label ...]

Checks the entire Django project for potential problems.
[...]
```

With your change: 
```sh
$ healthchecks help check
Unknown command: 'help check'. Did you mean check?
Type 'manage.py help' for usage.
```

Reply:
Then they are quoting wrong beforehand already. I am suing that pattern all over the place in my dotfiles.

Reply:
Yes, the outer shell does or did. Just pushed a proposal which is even uglier, but allows us keeping the quotes. Not sure if that's an improved compared to the version without quotes, but it seems to work.